source/distributed/grid_refinement.inst.in

```diff
@@ -59,8 +60,9 @@ namespace parallel
       refine_and_coarsen_fixed_number<deal_II_dimension-1,dealii::Vector<S>,deal_II_dimension>
       (parallel::distributed::Triangulation<deal_II_dimension-1,deal_II_dimension> &,
        const dealii::Vector<S> &,
-       const double,
-       const double);
+       double,
+       double,
```

Comment:
why remove the `const` here? Some compilers do check that it is there if it exists with the exception.